packages/core/src/components/widgets/BaseInput.js

```diff
@@ -61,19 +61,29 @@ function BaseInput(props) {
     return props.onChange(value === "" ? options.emptyValue : value);
   };
 
-  return (
+  return [
     <input
       className="form-control"
       readOnly={readonly}
       disabled={disabled}
       autoFocus={autofocus}
       value={value == null ? "" : value}
       {...inputProps}
+      list={schema.examples ? `examples_${inputProps.id}` : null}
```

Comment:
Can you make `idPrefix` be prepended to this `list` attribute and the `id` attribute in `<datalist>`?

Reply:
After looking briefly through the rest of the code, idPrefix seems to me, as far as I can tell, to only be used higher up, e.g. SchemaField, ObjectField, to generate an ID to pass on to lower level components. So for the BaseInput type, which seems to be the very lowest, this is property is not available, and would need to be passed on through a couple of layers, alongside the generated ID, for this to work. Are you sure you want this?

Reply:
Oh, I see. Looking at this code again, adding `idPrefix` seems to be unnecessary because `inputProps.id` will already include it.